File: CODES/1) C++/8) MINMAX/1.cpp
#include <bits/stdc++.h>

using namespace std;

int main(){
	int a, n, MAX, MIN;
	cout<<"n = "; cin>>n;
	for(int i=0; i<n; i++){
		cout<<i+1<<"-son = "; cin>>a;
		if(i==0){
			MAX=a;
			MIN=a;
		}
		if(MAX<a){
			MAX=a;
		}
		if(MIN>a){
			MIN=a;
		}
		if(i==n-1){
		cout<<"MIN = "<<MIN<<"   "<<"MAX = "<<MAX;
		}
	}
	return 0;
}

File: LANGUAGES/4) Php/8) MINMAX/1.php
<?php
$n = 10;
for ($i = 1; $i <= $n; $i++) {
    $a = rand() % 100;
    echo "A" . $i . " = " . $a . " <br>";
    if ($i == 1) {
        $max = $a;
        $min = $a;
    }
    if ($max < $a) {
        $max = $a;
    }
    if ($min > $a) {
        $min = $a;
    }
}
echo " max: " . $max . " <br>";
echo " min: " . $min . " <br>";
